android/package-lock.json

```diff
@@ -0,0 +1,9952 @@
+{
+    "name": "jest-appium-app-browserstack",
+    "version": "1.0.0",
+    "lockfileVersion": 3,
```

Comment:
This seems to be generated with latest node version. This can cause issues with customer setups where customers are using lower nodejs versions. 
Can we generate with Node 14/12 (the lowest node version that we support) ?